src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -9601,6 +9601,11 @@ public override Ast Copy()
         // Indicates that this ast was constructed as part of a schematized object instead of just a plain hash literal.
         internal bool IsSchemaElement { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether inline splatting syntax (@@) was used, false otherwise.
+        /// </summary>
+        public bool Splatted { get; set;  }
```

Comment:
Since `HashtableAst` inherits from `ExpressionAst`, we can always change that later without a breaking change. I think we should only increase the areas where it is exposed on a need by need basis unless you can think of a case where the change in this PR could make it useful to have the property on `ExpressionAst` as well.